### ISSUE TITLE
Hoff does not re-run tests unless there has been any other PRs merged

### ISSUE THREAD

**Issue by AlexCELewis** [channable/hoff#142]
If there is a flakey test on merging you should close and reopen the PR then try to remerge the PR. This should re-run the tests and will do so if it re-joins a queue. However, if there have been no merges in-between the failed flakey test and the re-run then hoff won't try to re-run the tests because the hash of the branch is the same so running the tests again is not triggered.

The fix is to make a slight modification to a commit that was pushed (e.g the commit message or an empty fixup) then re-run the hoff merge.

**Comment by fatho** [comment]
Yeah, with `retry` it should perform the entire integration step again. But I guess it would make sense for hoff to just redo the integration when prompted again, regardless of whether the "retry" shorthand or the full command was used. It certainly would be less confusing, I also ran into this at some point.
